mycli/packages/special/iocommands.py

```diff
@@ -193,3 +195,28 @@ def delete_favorite_query(arg, **_):
 
     return [(None, None, None, status)]
 
+@special_command('system', 'system [command]', 'Execute a system commmand.')
+def execute_system_command(arg, **_):
+    """
+    Execute a system command.
+    """
+    usage = "Syntax: system [command].\n"
+
+    if not arg:
+      return [(None, None, None, usage)]
+
+    try:
+        command = arg.strip()
+        if command.startswith('cd'):
+            result, error_message = handle_cd_command(arg)
+            if not result:
+                return [(None, None, None, error_message)]
```

Comment:
@amjith I think I've a better idea. I'll not add a `else` statement I'll just add the `return [(None, None, None, '')]` line instead. Is that okay? :smiley: